packages/components/src/components/tooltip/tooltip.hbs

```diff
@@ -15,20 +15,25 @@
     {{ carbon-icon 'Information16' }}
   </div>
 </div>
-{{!-- @todo add aria-hidden once JavaScript can toggle it --}}
-<div id="{{noHeading.idSuffix}}" data-floating-menu-direction="bottom" class="{{@root.prefix}}--tooltip"
-  data-avoid-focus-on-open role="dialog" aria-describedby="{{noHeading.idSuffix}}-body" aria-labelledby="{{noHeading.idSuffix}}-label">
-  <span class="{{@root.prefix}}--tooltip__caret"></span>
-  <p id="{{noHeading.idSuffix}}-body" >This is some tooltip text. This box shows the maximum amount of text that should appear inside. If more room is
-    needed
-    please use a modal instead.</p>
-  <div class="{{@root.prefix}}--tooltip__footer">
-    <a href="#" class="{{@root.prefix}}--link">Learn More</a>
-    <button class="{{@root.prefix}}--btn {{@root.prefix}}--btn--primary {{@root.prefix}}--btn--sm"
-      type="button">Create</button>
+<div class="{{@root.prefix}}--tooltip-wrapper">
```

Comment:
Same question to overflow menu - Would you put the trap element next to `<div class="bx--tooltip__footer">`?

Reply:
@asudoh the trap element needs to be outside of the content area of the tooltip not a sibling within it.  Currently with the way that the trap element works, it can not be a direct sibling of the content area which in the case of tooltips content is any direct child of `.bx--tooltip`

Current structure of tooltip
```
<bx--tooltip>
  <bx--tooltip__caret/>
  <p>Tooltip content</p>
  <bx--tooltip__footer/>
  <!-- trap element is a sibling of content if placed here and doesn't work -->
  <trap-element>
</bx--tooltip>
```

There are a couple different options:

**1. Scope the content area**

We could add a div within the tooltip around any of the content including the footer and update the event listener to watch the new div instead of the tooltip element

```
<bx--tooltip>
  <bx--tooltip__caret/>
  <div>
    <p>Tooltip content</p>
    <bx--tooltip__footer/>
  </div>
  <trap-element>
</bx--tooltip>
```

**2. Add a unique selector to the trap element**

We could give a unique selector to the trap element and watch for it to get focus and when it gets focus, immediate move it.  If that's the case we can keep the original HTML structure.

```
<bx--tooltip>
  <bx--tooltip__caret/>
  <p>Tooltip content</p>
  <bx--tooltip__footer/>
  <span tabindex="0" class="bx--focus-trap"></span>
</bx--tooltip>
```

Reply:
My apologies, I seems to have missed this comment. I think 1. makes sense. BTW instead of wrapper node ref, let's move the root node up to the wrapper as above structure tells. If we need to refer to where the root node used to be, we can introduce `options.selectorContent`. Thanks!